blueprints/in-repo-engine/files/lib/__name__/index.js

```diff
@@ -0,0 +1,8 @@
+/*jshint node:true*/
+module.exports = {
+  name: '<%= dasherizedModuleName %>',
+
+  isDevelopingAddon: function() {
+    return true;
+  }
+};
```

Comment:
Will update. When I was working locally I didn't notice any hinting errors being thrown, so it might have been fixed with more recent dependencies. Will double check that though.